back/src/main/java/com/speakerspace/config/FirebaseTokenRequest.java

```diff
@@ -0,0 +1,9 @@
+package com.speakerspace.config;
+
+public class FirebaseTokenRequest {
+    public String idToken;
+
+    public String getIdToken() {
+        return idToken;
+    }
+}
```

Comment:
todo: passe par un record, la syntaxe est plus courte, t'as un getter implicit (`firebaseTokenRequest.idToken()`), c'est immutable donc plus optimisé

```suggestion
public record FirebaseTokenRequest(String idToken) {}
```

Reply:
Globalement : pour tous les DTO, passe par des records par défaut et si tu bloques passe en class